code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -7,14 +7,16 @@
 //HALF_LIFE(0) -> Reaction completes immediately (default chems)
 //HALF_LIFE(1) -> Half of the reagents react immediately, the rest over the following ticks.
 //HALF_LIFE(2) -> Half of the reagents are consumed after 2 chemistry ticks.
-//HALF_LIFE(3) -> Half of the reagents are consumed after 3 chemistry ticks.
+//HALF_LIFE(3) -> Half of the reagents are consumed after 3 chemistry ticks. HALF LIFE 3 CONFIRMED?
 #define HALF_LIFE(ticks) (ticks? 1.0 - (0.5)**(1.0/(ticks*PROCESS_REACTION_ITER)) : 1.0)
 
 /datum/chemical_reaction
 	var/name = null
 	var/id = null
 	var/result = null
 	var/list/required_reagents = list()
+	var/list/required_temperatures_min = list() //Format: reagent name = required_kelvin. Temperatures must exceed this value to trigger.

```

Comment:
Ehh? Shouldn't it just be temperature for the entire container? All reagents will be at the same temperature when mixed, so it just seems like you're accounting for an impossible edge-case here.

Reply:
False. I want to create a system where you can mix 100 celcius iron with negative 100 celcius water to create a special reaction.